client/rest-high-level/src/main/java/org/elasticsearch/client/TasksRequestConverters.java

```diff
@@ -54,4 +56,16 @@ static Request listTasks(ListTasksRequest listTaskRequest) {
             .putParam("group_by", "none");
         return request;
     }
+
+    static Request getTask(GetTaskRequest getTaskRequest) {
+        String endpoint = new EndpointBuilder().addPathPartAsIs("_tasks")
+                .addColonSeparatedPathParts(getTaskRequest.getNodeId(), Long.toString(getTaskRequest.getTaskId()))
```

Comment:
I was thinking this couldve just been a 
```
.addPathPart(getTaskRequest.getNodeId() + ":" + Long.toString(getTaskRequest.getTaskId()))
```

since its technically one "path part" in a URL. Then there is no need for an extra method there. I was just trying to ensure we were using the EndpointBuilder in all the RequestConverters.

Reply:
OK. I saw a precedent for comma-delimited parts in EndPointBuilder and assumed colon-delimited would be another variant. I guess it's not that common a convention so I removed it.